src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,6 +362,216 @@ const formConfig = {
             }
           }
         },
+        // TODO: Should this be privateMedicalRecordsRelease or privateRecordsRelease? 
+        privateMedicalRecordsRelease: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-release',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          depends: (formData, index) => {
+            const hasRecords = _.get(`disabilities.${index}.view:privateMedicalRecords`, formData);
+            // TODO: enable once previous chapter merged
+            // const requestsRecords = !_.get(`disabilities.${index}.view:uploadPrivateRecords`, formData);
+            const requestsRecords = true;
+            return hasRecords && requestsRecords;
+          },
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:description': 'Please let us know where and when you received treatment. We’ll request your private medical records for you. If you have your private medical records available, you can upload them later in the application',
+                treatments: {
+                  'ui:options': {
+                    itemName: 'Record',
+                    viewField: () => {
+                      return (<div>hello</div>);
+                    }
+                  },
+                  items: {
+                    treatment: {
+                      'ui:order': ['treatmentCenterName', 'privateMedicalRecordsReleaseAccepted', 'view:privateMedicalRecordsReleasePermissionRestricted', 'startTreatment', 'endTreatment', 'treatmentCenterCountry', 'treatmentCenterStreet1', 'treatmentCenterStreet2', 'treatmentCenterCity', 'treatmentCenterState', 'treatmentCenterPostalCode', 'treatmentCenterPhone'],
```

Comment:
While you're doing your cleanup, could you break these onto multiple lines? Doesn't have to be one item per line, but something a little more manageable.

Reply:
Sounds good, updated.